### PR TITLE
feat: show full databases add owner field

### LINKED ISSUE #14153: feat: add owner field in SHOW FULL DATABASES

### PR DESCRIPTION
**Pull request by guojidan**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

add owner field in show databases

Fixes #14153 

## Tests

- [x] Unit Test
- [x] Logic Test
- [ ] Benchmark Test
- [ ] No Test  - _Explain why_

## Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/14184)

### CODE REVIEW

**Comment by guojidan** [comment]
@BohuTANG this pr can be reviewed

**Comment by TCeason** [comment]
Cc @flaneur2020  maybe it's not the right time to support show owners?

**Comment by flaneur2020** [comment]
@TCeason we need some further refactor about these logic, but I believe we can merge this PR first : )

**Comment by TCeason** [comment]
> @TCeason we need some further refactor about these logic, but I believe we can merge this PR first : )
> 
> 

How about support display owner field into show full databases? I think that's better.

Because as a user, if I can show this db, I don't care the db owner.

File: src/query/service/tests/it/servers/http/http_query_handlers.rs [hunk 1]
```diff
@@ -345,7 +345,7 @@ async fn test_show_databases() -> Result<()> {
     assert_eq!(status, StatusCode::OK, "{:?}", result);
     assert!(result.error.is_none(), "{:?}", result);
     // has two fields: catalog, name
```
Comment (TCeason) [inline src/query/service/tests/it/servers/http/http_query_handlers.rs:347]: The comment needed modify to three?

**Review by TCeason — APPROVED** [review]
LGTM, wait the ci passed. Better to modify the comment.

File: src/query/service/tests/it/servers/http/http_query_handlers.rs (hunk 1 above)
Comment (guojidan) [reply src/query/service/tests/it/servers/http/http_query_handlers.rs]: fixed : )

**Pull request merged**